### DISCUSSION TITLE
Better graphics performance after using AML?

### DISCUSSION THREAD

**Discussion by DoctorSteele** [X2CommunityCore/xcom2-launcher#292] [General]
I've noticed that since using AML, all the problems I used to have with graphical stuttering, etc., seem to have gone away.

Does this happen for anyone else?  I certainly am not complaining, but I'm at a loss to figure out how the two things could possibly be related.  I'm using the same actual mods (a few more, actually, because the Firaxis launcher couldn't recognize some of them) and haven't changed anything else that I'm aware of.

**Comment by RevZero** [comment]
Hmm, AML does not change anything related to graphics performance. It it likely caused by some change to your mod list after all. Maybe you reset you configs (-regenerateinis) or something? 
Did you experience slowdowns on the Geoscape only or also during tactical?

**Reply by DoctorSteele** [reply]
I don't think I ever used that argument, but I did use -nostartupmovies for the first time with AML, that's the only obvious change I can think of.

My slowdowns occurred both in Geoscape and tactical (seemed most common with Lost missions or ones with environmental effects like rain.)  My system's pretty high-end, so it never seemed likely to be related to hardware.

**Reply by RevZero** [reply]
Mod von conflicts CAN cause slowdowns. So if you have a high end system with respect to xcom system requirements, maybe there was a conflict that is now resolved for some reason.
